LoopKitUI/Views/LabelTextField.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LabelTextField.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-02-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct LabelTextField: View {
+    var label: String
+    var placeholder: String
+    @Binding var value: String
+    
+    public init(label: String, placeholder: String = "", value: Binding<String>) {
+        self.label = label
+        self.placeholder = placeholder
+        _value = value
+    }
+    
+    public var body: some View {
+        GeometryReader { geometry in
+            HStack(alignment: .firstTextBaseline) {
+                Text(self.label)
+                    .foregroundColor(.primary)
+                    .frame(width: geometry.size.width/2-5, alignment: .leading)
```

Comment:
Where does the special "-5" magic number come from?  Maybe consider putting it in. as a constant at the top of the struct?